readme2.md

```diff
@@ -0,0 +1,63 @@
+# insidious-bird
+
+## Description
+
+Make a flappy-bird clone in phaser io
+
+## Context
+
+Learn the ins and outs of Phaser by making a fun, interactive game.
+
+## Specifications
+
+- [ ] User can start game after front page loads
+- [ ] Player can control bird's movement to avoid obstacles
+- [ ] Player can reset/restart game at any time
+- [ ] Map is generated randomly, with various obstacles along the way
+- [ ] Player can pause game and save game
+- [ ] User log in and database to save high scores, achievements, flap count
+- [ ] Network for multiplayer functionality in a vs mode
+- [ ] Game saves win loss record after versus games
+- [ ] Add exploration mode where players can choose different paths by going up or down
+- [ ] Database saves a record of path so future players can try to find paths other players have taken for a bonus.
+- [ ] "Chimera mode" where various power ups change both your bird's appearance and how it interacts with obstacles
+- [ ] At end of "Chimera Mode" the game gives the option to save an image of your final bird with all powerups
+- [ ] Add shooter mode with enemy birds, weapons, and powerups to enhance your bird
+- [ ] Test backend and testable front end functions with chai
+
+## Stretch Goals
+- [ ] Test front end with selenium
```

Comment:
If you're using phaser, you're going to have a hard time testing the front end with selenium (or it's going to simply be a test that check is a file loads, which isn't a very meaningful test).  I strongly recommend the use of a unit testing framework for backend logic.